<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Sans Serif</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Monorotor Commander Desctop</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame_com">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QComboBox" name="comboBox_com">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_com">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>150</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Подключить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_com">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выберите COM-порт</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_model">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>521</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QTabWidget" name="tabWidget_model">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>491</width>
       <height>381</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <widget class="QWidget" name="tab_params">
      <attribute name="title">
       <string>Основные</string>
      </attribute>
      <widget class="QFrame" name="frame_params">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>481</width>
         <height>311</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_volume">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>10</y>
          <width>160</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Объём, мл</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_reverse">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>130</y>
          <width>160</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Реверс, мл</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_feedrate">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>70</y>
          <width>160</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Подача, мл/мин</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_dozators">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>180</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
       </widget>
       <widget class="MCDLineEdit" name="lineEdit_feedrate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>180</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
       </widget>
       <widget class="MCDLineEdit" name="lineEdit_reverse">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>180</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
       </widget>
       <widget class="MCDLineEdit" name="lineEdit_volume">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>180</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
       </widget>
       <widget class="QFrame" name="frame_ratio">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>180</y>
          <width>471</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="MCDLineEdit" name="lineEdit_ratioA">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>80</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
         <property name="maxLength">
          <number>15</number>
         </property>
        </widget>
        <widget class="MCDLineEdit" name="lineEdit_ratioB">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>10</y>
           <width>80</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
         <property name="maxLength">
          <number>15</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_ratioAB">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>10</y>
           <width>181</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Соотношение A:B</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_details">
      <attribute name="title">
       <string>Дополнительные</string>
      </attribute>
      <widget class="QFrame" name="frame_details">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>471</width>
         <height>241</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="MCDLineEdit" name="lineEdit_accel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>180</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_accel">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>10</y>
          <width>221</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ускорение, мл/минсек</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_gearA">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>70</y>
          <width>180</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Коэффициент А</string>
        </property>
       </widget>
       <widget class="MCDLineEdit" name="lineEdit_gearA">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>180</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
       </widget>
       <widget class="MCDLineEdit" name="lineEdit_gearB">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>180</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_gearB">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>130</y>
          <width>180</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Коэффициент B</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_dir">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>211</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Реверс направления</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_transmit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>461</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QPushButton" name="pushButton_start">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Пуск</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_continues">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Непрерывно</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_hold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>По удержанию</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_stop">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Стоп</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>461</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Вкладка</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Кнопки</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>331</width>
        <height>331</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton_nc">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>250</y>
         <width>80</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
      <widget class="MCDPushButton" name="pushButton_n9">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>170</y>
         <width>80</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
      <widget class="MCDPushButton" name="pushButton_n1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>80</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="MCDPushButton" name="pushButton_n3">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>10</y>
         <width>80</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="MCDPushButton" name="pushButton_n0">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>250</y>
         <width>80</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="MCDPushButton" name="pushButton_nd">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>80</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
      <widget class="MCDPushButton" name="pushButton_n2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>80</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="MCDPushButton" name="pushButton_n4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>80</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="MCDPushButton" name="pushButton_n7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>80</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
      <widget class="MCDPushButton" name="pushButton_n5">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>80</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="MCDPushButton" name="pushButton_n8">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>80</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
      <widget class="MCDPushButton" name="pushButton_n6">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>90</y>
         <width>80</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Лог</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_log">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>431</width>
        <height>331</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFirmware">
    <property name="title">
     <string>MC Firmware</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Обновление</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action_MONOROTOR"/>
    <addaction name="action_6"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuFirmware"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Открыть репозиторий</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Скачать</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Прошить</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Открыть репозиторий</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Обновить</string>
   </property>
  </action>
  <action name="action_MONOROTOR">
   <property name="text">
    <string>О нас</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MCDLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mcdlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>MCDPushButton</class>
   <extends>QPushButton</extends>
   <header>mcdpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
